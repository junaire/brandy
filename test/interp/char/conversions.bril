@main() {
  c1: char = const 'A';
  c2: char = const '𝛀'; 
  i1: int = const 65;
  i2: int = const 120512;

  c3: char = int2char i1;
  i3: int = char2int c1;
  c4: char = int2char i2;
  i4: int = char2int c2;

  print c3 i3 c4 i4;
}