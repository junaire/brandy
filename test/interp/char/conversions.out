A 65 𝛀 120512
